backend/notifications/models.py

```diff
@@ -29,6 +29,10 @@ class EmailTemplateIdentifier(models.TextChoices):
 
     reset_password = "reset_password", _("[System] Reset password")
 
+    grant_application_confirmation = (
```

Comment:
Do you think this name is clear enough?

Reply:
any better ideas?

Reply:
can't think of a better name, "application" is what thrown me off a bit but it is not a big deal